<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="bufferProgressBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 10px;
	padding-right: 1px;
}

QProgressBar::chunk {
    background-color:#19BE5E;
	width: 72px;
	border-radius: 10px;
    margin: 1px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="productorBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color: rgb(136, 136, 136);
	border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="productorOn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(14, 216, 31);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="productorOff">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(255, 0, 0);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Productor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Consumidor</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="bufferBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color: rgb(136, 136, 136);
	border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="bufferOn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(14, 216, 31);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="bufferOff">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(255, 0, 0);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="iniciar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="productorBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color: rgb(136, 136, 136);
	border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="consumidorOn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(14, 216, 31);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="consumidorOff">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:  rgb(255, 0, 0);
	border: none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
